src/main/java/io/github/genomicdatainfrastructure/discovery/utils/PackageShowMapper.java

```diff
@@ -44,16 +42,75 @@ public RetrievedDataset from(CkanPackage ckanPackage) {
                 .url(ckanPackage.getUrl())
                 .languages(values(ckanPackage.getLanguage()))
                 .contact(value(ckanPackage.getContactUri()))
-                .hasVersions(values(ckanPackage.getHasVersion()))
                 .accessRights(value(ckanPackage.getAccessRights()))
-                .conformsTo(values(ckanPackage.getConformsTo()))
                 .provenance(ckanPackage.getProvenance())
                 .spatial(value(ckanPackage.getSpatialUri()))
                 .distributions(distributions(ckanPackage))
                 .keywords(keywords(ckanPackage))
+                .contacts(contactPoints(ckanPackage.getContacts()))
+                .datasetRelationships(relations(ckanPackage.getDatasetRelationships()))
+                .dataDictionary(dictionary(ckanPackage.getDataDictionary()))
                 .build();
     }
 
+    private List<ContactPoints> contactPoints(List<CkanContactPoints> values) {
+        return ofNullable(values)
+                .orElseGet(List::of)
+                .stream()
+                .map(PackageShowMapper::contactPoint)
+                .filter(Objects::nonNull)
```

Comment:
important: move `.filter(Objects::nonNull)` to like 60, so you don't need to check again if the object in null in `PackageShowMapper::contactPoint`.